daemons/attrd/attrd_corosync.c

```diff
@@ -362,7 +363,7 @@ attrd_peer_update_one(crm_node_t *peer, xmlNode *xml, bool filter)
 }
 
 static void
-broadcast_unseen_local_values(crm_node_t *peer, xmlNode *xml)
+broadcast_unseen_local_values(xmlNode *xml)
```

Comment:
The `xml` argument isn't used in the function

Reply:
Thanks